launch/docker.go

```diff
@@ -19,8 +19,11 @@ var _ runner = (*docker)(nil)
 var execCommand = exec.Command
 
 const (
-	artifactsDir = "artifacts"
-	logFile      = "builds.log"
+	ArtifactsDir = "artifacts"
+	LogFile      = "builds.log"
+	// The definition of "ScmHost" and "OrgRepo" is in "PipelineFromID" of "screwdriver/screwdriver_local.go"
+	ScmHost = "screwdriver.cd"
+	OrgRepo = "sd-local/local-build"
```

Comment:
Are there any reasons that these variable are public?

Reply:
thx.
I fixed it.